### DISCUSSION TITLE
feature: Link map changes

### DISCUSSION THREAD

**Discussion by Arttroy** [AscensionGameDev/Intersect-Engine#1474] [Ideas]
**Is your feature request related to a problem? Please describe.**

A world map like this should be able to wrap-around facilitating a "round" world.

![image](https://user-images.githubusercontent.com/1476550/166058091-adb72340-b404-4412-b9b2-ebef69af4f6b.png)

**Describe the solution you'd like**
Edge maps should be able to connect to opposite-edge maps to create a "round" world.

**Comment by blinkuz** [comment]
I think this is an exaggeratedly specific functionality for your type of project, which is why I don't think it will be accepted, the description you propose is also not very understandable, maybe if you are a little more specific it can be considered, but I still think it is something very specific.

**Comment by Arttroy** [comment]
I'm not english so i do with what i know sorry ^^ I don't know how to explain more the situation but if you can explain me which precisions you want i can add it.

**Comment by pandinocoder** [comment]
While yes the request could be worded better, I specifically asked him to open this feature request and personally like the idea. With some reasonable limitations I do believe wrapping like this should be allowable.

**Comment by Arttroy** [comment]
I don't know how to do it, but maybe it's possible to allow linking with more than one map so it should be done without too much code changes. Not sure, as i say i know few things about ruby scripting but c# is completely a new world for me. Just one more link possibility will do the thing.

Or else a option that allow to link automatically all the edges left/right and upper/lower if turned on. So we should have the choice to do it or not.

These are ideas i have no clue it's possible to do such things.

**Comment by Arttroy** [comment]
Hello all, i'm on my way to create my game and now i've thought of something that will slightly complicate things... In the case someone use this feature, he will need to create maps for interiors such as house, temple, caves etc... So the system as described won't allow to use others maps than the ones linked before...
I think it would be complicate to code something that allow to select which ones are linked and which are not. Probably the best thing to do is to allow maps to be linked to more than one.

> make it so it so they warp players to the opposite edge

I already know i can do that, on rpgmaker this was the way to do such thing
